docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -69,8 +71,16 @@ import TabIndicator from '@material-ui/core/Tabs/TabIndicator';
 
 ### Option 2
 
-This option provides the best DX and UX.
-However, you need to apply the following steps correctly.
+This option provides the best User Experience and Developer Experience:
+
+- UX: the Babel plugin makes it as fast as the option 1.
```

Comment:
```suggestion
- UX: The Babel plugin makes it as fast as option 1.
```

If the performance (or rather the bundle size) is the same as option 1, how us the UX "the best" with this option?

Reply:
Great point. I'm changing it to:
```diff
-This option provides the best User Experience and Developer Experience:
+This option provides the best User Experience and Developer Experience tradeoff:
```

Reply:
This change makes it again unclear.
If someone wants to have the fastest imports, then this change again confuses it to being labeled as just a 'tradeoff', and justifies promoting using direct imports (and supporting it with a codemode)

Reply:
Oh, you made me realize that the wording is wrong. These are two DX pros, the UX pros are about forcing top-level tree shaking for bundler that doesn't support it. Maybe it's marginal and we should only focus on the DX tradeoff. The tradeoff is the need for a Babel configuration step to get fast startup time in exchange.

To answer your concern, isn't the list below making it clear?